tdrs-backend/tdpservice/data_files/models.py

```diff
@@ -206,6 +207,10 @@ def submitted_by(self):
         """Return the author as a string for this data file."""
         return self.user.get_full_name()
 
+    def admin_link(self):
+        """Return a link to the admin console for this file."""
+        return f"{settings.FRONTEND_BASE_URL}/admin/data_files/datafile/?id={self.pk}"
```

Comment:
you shouldn't need to hardcode the whole URL, you should either use reverse form django or reverse_actions from DRF as stated here: https://www.django-rest-framework.org/api-guide/viewsets/#reversing-action-urls